### PR TITLE
MLDB-1205 make sure we skip newlines in sql_expression

### PR DESCRIPTION
**Pull request by steadtler**

### CODE REVIEW

File: sql/sql_expression.cc [hunk 1]
```diff
@@ -1119,8 +1119,10 @@ parse(ML::Parse_Context & context, int currentPrecedence, bool allowUtf8)
             lhs = arg;  
         }
 
-        context.skip_whitespace();
+        cerr << "test1" << endl;
```
Comment (guyd) [inline sql/sql_expression.cc]: debug code

File: sql/sql_expression.cc [hunk 2]
```diff
@@ -1119,8 +1119,10 @@ parse(ML::Parse_Context & context, int currentPrecedence, bool allowUtf8)
...
+        skip_whitespace(context);
         context.expect_literal('}');
+        cerr << "test2" << endl;
```
Comment (guyd) [inline sql/sql_expression.cc]: here also

**Comment by guyd** [comment]
+1

File: sql/sql_expression.cc (hunk 1 above)
Comment (steadtler) [inline sql/sql_expression.cc]: the hell, I removed that before commit. Git is wierd sometimes

**Pull request merged**